### PR TITLE
feat(resource): adding resource for eventbridge pipes

### PR DESCRIPTION
**Pull request by MSchouwPablo**

### CODE REVIEW

**Comment by MSchouwPablo** [comment]
@ekristen hi! added support for eventbridge pipes, but I didn't format my title correctly on the first try, and I think the commit-link check is not picking up new changes? lmk if i can fix it :)

File: resources/pipes-pipe.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	"github.com/aws/aws-sdk-go/service/pipes"
+
+	"github.com/ekristen/libnuke/pkg/registry"
+	"github.com/ekristen/libnuke/pkg/resource"
+	"github.com/ekristen/libnuke/pkg/types"
+
+	"github.com/ekristen/aws-nuke/v3/pkg/nuke"
+)
+
+const PipesPipeResource = "PipesPipes"
```
Comment (ekristen) [inline resources/pipes-pipe.go:16]: We want the resource type to be singular. The plural on the service is fine, we usually try to make that match.

```suggestion
const PipesPipeResource = "PipesPipe"
```

**Review by ekristen — CHANGES_REQUESTED** [review]
Overall great PR! Thanks! You got all the formatting correct, even the imports, thanks!You also picked up some of the newer tooling around properties! 

One minor change around the resource name.

**Comment by MSchouwPablo** [comment]
Ah great, thanks. Included your change :)

**Comment by ekristen** [comment]
@MSchouwPablo I did a force update on your branch to fix the commits so commit-lint would pass, however I should have asked before I did. If you'd like to force push your own branch so you retain full author on your commits, please do, otherwise let me know and I'll merge. I can provide rebase instructions should you need them. 

Either way I'm good. If you are ok with it as is, let me know and I'll just merge.

**Review by ekristen — APPROVED** [review]

**Pull request merged**